macros/anaUltraScurve.py

```diff
@@ -115,6 +115,8 @@
     myT.Branch( 'scurve_h', scurve_h)
     chi2 = array( 'f', [ 0 ] )
     myT.Branch( 'chi2', chi2, 'chi2/F')
+    ndf = array( 'f', [ 0 ] )
```

Comment:
It's mostly about correctly describing things, as the performance hit of doing things "wrong" probably won't be noticed.
Things that are integer types should probably be stored as integer types.
In fact, looking now, I see there are other instances of using```float``` when ```int``` should be preferred (```Nev```, which changes types between input tree and output tree)